templates/repo/header.tmpl

```diff
@@ -55,6 +52,12 @@
 							</div>
 						</form>
 					{{end}}
+					{{if $.EnableFeed}}
+					{{/* An extra div-element is not necessary here, as this button does not secretly contain two buttons. */}}
+					<button class="ui compact small basic button" data-url="{{$.RepoLink}}.rss" data-tooltip-content="{{ctx.Locale.Tr "rss_feed"}}">
```

Comment:
Again, has this code been tested?

I do not think `data-url` would work for a RSS link.

Reply:
Gosh, I got so preoccupied with trying to make the icons look consistent enough UI-wise to the point where I may have overlooked the possibility that such an attempt could possibly break things like that.

I got super happy that I managed to get a Gitea change in without a change requested or breaking things - I understand that this isn't the first time. I will spend the next few minutes looking into this, issuing a fix and if I don't manage it, I'll send in a Revert and then start over.

Reply:
I think you could try to use `<a class="...." href="{{the RSS link}}">...</a>`.

`<button>` means it requires extra JS code to work with a link or it just submits a form.

`link-action` means it is an AJAX POST request by `data-url`.

Reply:
I don't want to take more of your time, so I'll inform you of the following: The faulty behavior seems to come from a botched very-early-in-the-morning rebase. I'll need like 30 minutes to properly fix this and then send a subsequent review that fixes this.

Reply:
> I think you could try to use <a class="...." href="{{the RSS link}}">...</a>.

Yep, that was my plan. I just made a commit that does this exactly (and explicitly defines `role=button` so that it works under screen readers), but I'll need a few extra minutes to fully test this.

In the future, I'll try to send videos after deciding that the commit is PR ready for self-verification reasons and as well as to ease the review process.